public/webpackShims/vis.js

```diff
@@ -1,2 +1,2 @@
 require('../../node_modules/vis/dist/vis.min.css');
-module.exports = require('../../node_modules/vis/dist/vis.min.js');
```

Comment:
To be honest it is a left over after a debugging session 
But I would leave it as in this way
1 it is easier to debug 
2 webpack will optimize anyway (if we leave the min I think you have to tell webpack to skip such a file during optimization)